src/components/Settings/AccountTab.react.js

```diff
@@ -250,6 +250,14 @@ class AccountTab extends React.Component {
   handleSubmit = async () => {
     const { timeZone, prefLanguage, userName, avatarType } = this.state;
     const { actions, userEmailId } = this.props;
+    if (!(userName.length > 0)) {
```

Comment:
Use `userName.trim().length` to remove blank spaces

Reply:
changed it !